apps/pie-docs/src/components/modal/overview/overview.md

```diff
@@ -0,0 +1,424 @@
+    alt: "Default variation of modal.",
+    width: 600
+} %}
+
+---
+
+## Modifiers
+
+### Header
+
+#### Back
+
+If a multi-step Modal is required, from steps two onwards a back Icon Button is displayed allowing the user to move back to the preview view.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-header-back.png",
+    alt: "A modal with a back icon button in the header.",
+    width: 600
+} %}
+
+#### Close
+
+Use this Header modifier when if the modal can't be dismissed.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-header-close.png",
+    alt: "A modal with a close icon button in the header.",
+    width: 600
+} %}
+
+### Footer
+
+You can replace the footer Buttons with any Button or Button pair defined within the Button guidance.
+
+#### Double
+
+A maximum of two Buttons are allowed within the Modal’s footer, with the primary Button right aligned and the lower emphasis Button on the left.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-double.png",
+    alt: "A modal with a close icon button in the header.",
+    width: 600
+} %}
+
+#### Single
+
+You can use the Passive modifier for the footer in instances where the user doesn't need to take an action. This will remove the buttons at the bottom of the Modal.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-single.png",
+    alt: "A modal with a pair of primary and ghost buttons in its footer.",
+    width: 600
+} %}
+
+#### Passive
+
+A single Button variant is available when only one action is required.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-passive.png",
+    alt: "A modal with a single primary button in its footer.",
+    width: 600
+} %}
+
+---
+
+#### Sizes
+
+There are three responsive Modal sizes: large, medium and small. Choose a size that works best for the amount of Modal content you have.
+
+Modals with short messages should use the small Modal to avoid long single lines; for complex or larger content, medium or large Modal will be more suitable.
+
+{% componentDetailsTable {
+  tableData: sizes,
+  priority: 2
+} %}
+
+### Large
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-large.png",
+    alt: "Large size modal.",
+    width: 968
+} %}
+
+### Medium
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-medium.png",
+    alt: "Medium size modal.",
+    width: 520
+} %}
+
+### Small
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-small.png",
+    alt: "Small size modal.",
+    width: 336
+} %}
+
+---
+
+## Narrow
+
+### Sizes
+
+#### Full screen
+
+The Wide large and optionally medium size, has the corresponding responsive option of fullscreen at narrow.
```

Comment:
This sentence doesn't read very naturally to me. Is it trying to say that large modals are fullscreen at narrow viewports, and medium modals have the option of being fullscreen at narrow viewports?

Reply:
@nazanin

Reply:
For reference Naz, the 75% is 
'The Wide small and optionally medium size, has the corresponding responsive option of 75% width at narrow.'

Reply:
We have 2 terms for describing screen sizes in component structure, wide for desktop screens and narrow for mobile and smaller ones, It's saying that a modal (large size and potentially medium) turns into a full screen modal on mobile screens, and smaller size modals will 75% with of the whole screen, I'm not the native to reword this :D

Reply:
Thanks I'll try and updated :)

Reply:
I have gone for 
Large modals will expand to fill the entire page width at narrow viewports.
and
Medium and small modals will expand to 75% page width at narrow viewports.
What do you think?